### ISSUE TITLE
could not compile it

### ISSUE THREAD

**Issue by tXJtqlb3bbeKD1yW8SV7** [SofusA/roslyn-language-server#6]
Windows 10 using `cargo install --git https://github.com/SofusA/roslyn-language-server`
```
error[E0432]: unresolved import `tokio::net::UnixStream`
  --> src/main.rs:9:5
   |
9  |     net::UnixStream,
   |     ^^^^^^^^^^^^^^^ no `UnixStream` in `net`
   |
note: found an item that was configured out
  --> C:\Users\default.LAPTOP-E48G1MLG\.cargo\registry\src\index.crates.io-6f17d22bba15001f\tokio-1.39.3\src\net\mod.rs:57:27
   |
57 |     pub use unix::stream::UnixStream;
   |                           ^^^^^^^^^^

For more information about this error, try `rustc --explain E0432`.
error: could not compile `roslyn-language-server` (bin "roslyn-language-server") due to 1 previous error
error: failed to compile `roslyn-language-server v0.1.0 (https://github.com/SofusA/roslyn-language-server#3efaf3c9)`, intermediate artifacts can be found at `C:\Users\DEFAUL~1.LAP\AppData\Local\Temp\cargo-installa0G1Cb`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.
```

**Comment by kurushimee** [comment]
It doesn't support Windows yet

**Comment by SofusA** [comment]
Closed in favour of #7

**Comment by SofusA** [comment]
Solved by #8
